pyteal/ast/method.py

```diff
@@ -0,0 +1,43 @@
+from typing import TYPE_CHECKING
+from pyteal.errors import TealInputError
+
+from pyteal.types import TealType
+
+from ..types import TealType
+from ..ir import TealOp, Op, TealBlock
+from .leafexpr import LeafExpr
+
+if TYPE_CHECKING:
+    from ..compiler import CompileOptions
+
+
+class Method(LeafExpr):
```

Comment:
Renaming this to `MethodSignature` would probably be more clear. PyTeal may want to use `Method` to refer to a method definition in the future